tests/unit_test/callback/pyscript_handler_test.py

```
@@ -1224,6 +1370,106 @@
+            "pincode": 538494
+        }
+    }
+
+    mock_document = MagicMock()
+    mock_document.to_mongo.return_value.to_dict.return_value = mock_data
+
+    mock_queryset = MagicMock()
+    mock_queryset.first.return_value = mock_document
+
+    with pytest.raises(Exception, match="Missing collection name"):
+        result = PyscriptSharedUtility.get_crud_metadata(collection_name="", bot='test_bot', user='test_user')
```

Ruff (0.11.9)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)